djangorestapi/api/tests.py

```diff
@@ -17,3 +21,33 @@ def test_model_can_create_a_todo_list(self):
         self.todo_list.save()
         new_count = ToDoList.objects.count()
         self.assertNotEqual(initial_count, new_count)
+
+
+class TestUpdateList(TestCase):
+    """Test for updating list items"""
+    
+    # define variable needed to run test
+    def setUp(self):
+        self.client = APIClient()
+        self.update_data = {'item_name': 'change item'}
+        self.res = self.client.post(reverse('create'),self.update_data,format='json')
```

Comment:
Good work so far, but I would recommend that you use descriptive variable names. i.e instead of using (res) you can use response